src/pages/Route/service.ts

```diff
@@ -50,8 +50,8 @@ export const fetchList = ({ current = 1, pageSize = 10 }, search: string) => {
 };
 
 export const remove = (rid: number) => request(`/routes/${rid}`, { method: 'DELETE' });
-export const offline = (rid: number) => request(`/offlineroutes/${rid}`, { method: 'PUT' });
-export const publish = (rid: number) => request(`/publishroutes/${rid}`, { method: 'PUT' });
+export const offline = (rid: number) => request(`/routes/${rid}/offline`, { method: 'PUT' });
+export const publish = (rid: number) => request(`/routes/${rid}/publish`, { method: 'PUT' });
```

Comment:
LGTM, and it seems to need a new PR on manager api.

Reply:
done